client/components/premium-global-styles-upgrade-modal/style.scss

```diff
@@ -68,6 +68,38 @@
 		}
 	}
 
+	.upgrade-modal__plan {
+		margin-bottom: 12px;
+
+		.upgrade-modal__plan-heading {
+			margin-bottom: 8px;
+			font-size: 20px; /* stylelint-disable-line declaration-property-unit-allowed-list */
```

Comment:
Is there a reason for not using [variables](https://github.com/Automattic/wp-calypso/blob/d048aa21754d7ded723158163d584d056afaa0d2/packages/typography/styles/variables.scss) for these font sizes?

E.g.

```diff
- font-size: 20px;
+ font-size: $font-title-small;

- font-size: 32px;
+ font-size: $font-title-large;

- font-size: 16px;
+ font-size: $font-body;
```

It should also take care of the Stylelint comments.

Reply:
Oops, thanks for pointing it out. I copied the style from `client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-upsell.scss` 😅 Addressed on https://github.com/Automattic/wp-calypso/pull/86057/commits/41e545fb78a24c8f58b79420102936de15d7587f